spec/spec_helper.rb

```diff
@@ -64,6 +67,10 @@ def versioned_kv_by_default?
   Gem::Requirement.new(">= 0.10").satisfied_by?(TEST_VAULT_VERSION)
 end
 
+def vault_is_enterprise?
+  vault_version_string.match(/\+i(ent|prem)$/)
```

Comment:
Can you remind me what the `i` does here?

Reply:
That... is probably a mistake from my vim using. I'll adjust it.

Reply:
The idea is that it should match `+ent` and `+prem`, I probably accidentally added it when trying to type something.